dialects/postgresql/src/main/kotlin/app/cash/sqldelight/dialects/postgresql/PostgreSqlType.kt

```diff
@@ -32,7 +33,7 @@ internal enum class PostgreSqlType(override val javaType: TypeName) : DialectTyp
           BIG_INT -> "bindLong"
           DATE, TIME, TIMESTAMP, TIMESTAMP_TIMEZONE, INTERVAL, UUID -> "bindObject"
           NUMERIC -> "bindBigDecimal"
-          JSON -> "bindObjectOther"
+          JSON, TSVECTOR -> "bindObjectOther"
```

Comment:
TSVECTOR -> "bindObjectOther" will have to be updated after https://github.com/cashapp/sqldelight/pull/5098 is merged 🧜